lib/coins/ada.js

```diff
@@ -0,0 +1,41 @@
+const b58 = require('b58')
+const cbor = require('cbor-sync')
+const CRC32 = require('crc-32'); 
+
+module.exports = {depositUrl, parseUrl}
+
+function parseUrl(network, address) {
+  console.log('DEBUG16: [%s] *%s*', network, address)
+
+  // Cardano in Byron era not recognize network type (main/test) by address prefixes like f.e. bitcoin
+  if (!validate(address)) return null
+
+  return address
+}
+
+function depositUrl(address, amount) {
+  return `${address}?amount=${amount}`
+}
+
+function validate(address) { 
+  const bytes = b58.decode(address)
```

Comment:
What happens if an address is not in base58? Would not it throw some exception?

Reply:
I will add try/catch blocks and return false